ruby/ql/lib/codeql/ruby/regexp/internal/DebugRegExpConfiguration.ql

```diff
@@ -1,12 +0,0 @@
-/**
- * @description Used to debug the discovery of regexp literals.
- * @kind path-problem
- */
-
-import RegExpConfiguration
```

Comment:
Let's keep this query. It would be interesting to see the DCA diff of this query when switching from dataflow to type tracking.

Reply:
I can't keep it as a `@kind path-problem`, but I can rewrite it a bit to get the same result.